test/katex-spec.js

```diff
@@ -116,6 +116,12 @@ describe("A punct parser", function() {
     });
 });
 
+describe("A \\colon builder", function() {
+    it("should not fail", function() {
+        expect("a \\colon b").toBuild(strictSettings);
+    });
+});
+
```

Comment:
I'm not sure whether we should have seperate test for a macro.

Reply:
FWIW, the "macro expander" section of tests has a few tests along these lines. But probably the screenshot test is more important here.